core-bundle/src/Image/Studio/Studio.php

```diff
@@ -42,19 +44,56 @@ public function createFigureBuilder(): FigureBuilder
     }
 
     /**
+     * @param string|ImageInterface                      $filePathOrImageInterface
      * @param array|PictureConfiguration|int|string|null $sizeConfiguration
      */
-    public function createImage(string $filePath, $sizeConfiguration): ImageResult
+    public function createImage($filePathOrImageInterface, $sizeConfiguration): ImageResult
```

Comment:
I think `$filePathOrImage` would be better without the `Interface` ☺️

Reply:
Agreed.